entries/l/lego.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "LEGO": {
+    "domain": "lego.com",
+    "url": "https://www.lego.com/en-us",
```

Comment:
```suggestion
```

You only need the `url` key if going to the base domain, that is `https://[domain]`, fails

Reply:
Thank you for the explanation.